source/use-cases/sensitive-data-encryption.txt

```diff
@@ -57,12 +57,220 @@ cubilia Curae; Nunc non interdum purus, ultricies laoreet tortor.
 B. Define a JSON Schema
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Proin non mi a felis luctus vulputate nec pharetra odio. Ut pretium
-scelerisque nulla, sed euismod nisi pellentesque sit amet. Proin mollis
-mauris eu libero ullamcorper rutrum. Pellentesque vitae enim sed magna
-accumsan maximus et quis mi. Ut vel laoreet turpis. Vivamus vitae odio
-sed arcu finibus pretium vel maximus nisi. Nullam gravida consequat
-porta.
+MongoDB uses JSON Schema to validate data that is written to a collection.
```

Comment:
Let's link this to https://docs.mongodb.com/manual/reference/operator/query/jsonSchema/#json-schema

Reply:
Done